orchestra/templates/communication/available_staffing_requests.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'orchestra/base.html' %}
+{% load staticfiles %}
+
+
+{% block title %} Available Tasks {% endblock %}
+
+{% block main %}
+<div class="container-fluid main-content staffing-container">
```

Comment:
will users be frustrated if the table is out of date if tasks are quickly picked up?